CIP-0058/README.md

```diff
@@ -7,7 +7,7 @@ Authors:
     - Maximilian König <maximilian@mlabs.city>
 Implementors:
     - Las Safin <me@las.rs>
-Status: Proposed
+Status: Inactive (superceded by CIP-0121 and CIP-0122)
```

Comment:
```suggestion
Status: Inactive (superseded by CIP-0121 and CIP-0122)
```
https://www.merriam-webster.com/dictionary/supersede

Reply:
I _always_ get that one wrong, thanks!